docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,304 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5a41c12-3002-4691-930f-dac5214443bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def rep_dl(s):\n",
+    "    return os.getcwd()+'/archive/img_align_celeba/img_align_celeba/'+s\n",
+    "\n",
+    "data2 = np.array([rep_dl(i) for i in df.index])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2b71fa0-ede8-4d16-8f13-da8d98acc90e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_label_name(ls):\n",
+    "    return [class_names[i] for i in ls]\n",
+    "\n",
+    "pred_label_names = [get_label_name(i) for i in pred_labels]\n",
+    "label_names = [get_label_name(i) for i in multi_labels]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fffa88f6-84d7-45fe-8214-0e22079a06d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = 10\n",
+    "h = 10\n",
+    "fig = plt.figure(figsize=(32, 32))\n",
+    "columns = 1\n",
+    "rows = 20\n",
+    "ct=0\n",
+    "for i in range(1, columns*rows +1):\n",
+    "    pos = issues[ct]\n",
+    "    label_error = label_names[pos]\n",
+    "    \n",
+    "    path = data2[pos]\n",
+    "    img = torchvision.io.read_image(path).swapaxes(0,2).swapaxes(0,1)\n",
+    "    fig.add_subplot(rows, columns, i)\n",
+    "    plt.axis('off')\n",
+    "    plt.imshow(img)\n",
+    "    plt.text(200, 50, data2[pos].split('/')[-1], bbox=dict(fill=False, edgecolor='black', linewidth=1))\n",
+    "    plt.text(200, 100, label_error, bbox=dict(fill=False, edgecolor='red', linewidth=1))\n",
+    "    plt.text(200, 150, pred_label_names[pos], bbox=dict(fill=False, edgecolor='green', linewidth=1))\n",
+    "    ct+=1\n",
+    "plt.savefig(\"errors.pdf\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d65af827-aeda-4b6b-9ae7-b1f0b84700d5",
+   "metadata": {},
+   "source": [
+    "## Have labels formatted as a one-hot (multi-hot) binary matrix?\n",
+    "For multi-label classification, cleanlab expects labels to be formatted as a list of lists, where each entry is an integer corresponding to a particular class. Here are some functions you can use to easily convert labels between these two formats. \n",
+    "```python\n",
+    "from cleanlab.internal.multilabel_utils import int2onehot\n",
+    "\n",
+    "y_binary_matrix = int2onehot(labels, K=num_classes)\n",
+    "```"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.12"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Don't worry about polishing the text too much, I'll do a rewrite of the text for you to make the tutorial as clear as possible